Automattic/sensei
# Show admin notice if Sensei hasn't been built correctly

Fixes #3599 

### Changes proposed in this Pull Request

* Add warning when assets/dist directory isn't present

### Testing instructions

* Install and configure fresh WordPress
* Clone sensei repository to the plugins directory.
* Log in to the admin panel and go to the Plugins page.
* Click "Activate" for Sensei LMS
* You'll see a warning as it is on the attached screenshot
* Run `npm install & composer install & npm run build:assets`
* Reload the page: the warning disappears


### Screenshot / Video
<img width="703" alt="Screenshot 2020-11-12 at 17 47 43" src="https://user-images.githubusercontent.com/329356/98955288-c2ed4980-250f-11eb-8f10-8636089c8ace.png">
